src/ui/buttons/util.js

```diff
@@ -39,3 +39,20 @@ export function calculateMessagePosition(
   }
   return MESSAGE_POSITION.BOTTOM;
 }
+
+export function checkLabelEligibility(
+  label?: $Values<typeof BUTTON_LABEL>,
+  buyerCountry: $Values<typeof COUNTRY>
+): $Values<typeof BUTTON_LABEL> | typeof undefined {
+  const eligibleCountriesForInstallmentLabel = ["BR", "MX"];
+
+  // Don't render the installment label if buyerCountry is not eligible for installment product
+  if (
+    label === BUTTON_LABEL.INSTALLMENT &&
+    !eligibleCountriesForInstallmentLabel.includes(buyerCountry)
+  ) {
+    return BUTTON_LABEL.PAYPAL;
```

Comment:
@wsbrunson Do we want to add any logging here if the label is not eligible?

Reply:
I don't think so, it should be normal for a merchant to be using Installments but have buyers from non-eligible countries visit their site